src/top_model/main.cpp

```diff
@@ -1,6 +1,8 @@
 
 #define TOP_MODEL_OUTPUT "data/abp_output_0.txt"
```

Comment:
Since the program is to be executed from bin folder, from the command prompt, the expected output path will be  **"../data/abp_output_0.txt"**,.
Please try executing from command prompt and verify.
You may have to check the same for all macro defined output paths in all files